src/controllers/resize-controller.js

```diff
@@ -68,9 +74,7 @@ export class ResizeController {
     this._host.requestUpdate();
 
     if (this._callback != null) {
-      this._host.updateComplete.then(() => {
-        this._callback();
-      });
+      this._callback(this.width);
```

Comment:
nitpick: you could use the internal property instead of the getter
```suggestion
this._callback(this._width);
```